src/components/TodosContext.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { useEffect, useState } from 'react';
+import { Status } from '../types/Status';
+import { TodoContextProps } from '../types/TodosContextProps';
+import { getTodos } from '../api/todos';
+import { Todo } from '../types/Todo';
+// eslint-disable-next-line import/no-cycle
+import { USER_ID } from '../App';
+
+export const TodoContext = React
+  .createContext<TodoContextProps>({
+  todos: [],
+  filter: Status.ALL,
+  addTodo: () => {},
+  toggleCompleted: () => {},
+  toggleAllCompleted: () => {},
+  deleteTodo: () => {},
+  clearCompleted: () => {},
+  updateTodoTitle: () => {},
+  setFilter: () => {},
+  errorMessage: '',
+  setErrorMessage: () => {},
+  handlerDeleteCompleted: () => {},
+});
+
+type Props = {
```

Comment:
Use interface prefer.
Type has unique ability but if you don`t need them use interface. It is sampled in scaled and has more interesting features.